rest/resource-server/src/main/java/org/eclipse/sw360/rest/resourceserver/clearingrequest/ClearingRequestResourceProcessor.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright Healthineers 2022. Part of the SW360 Portal Project.
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
```

Comment:
please add new line after each ... well, the file should look something like this: [ComponentResourceProcessor.java](https://github.com/eclipse/sw360/blob/main/rest/resource-server/src/main/java/org/eclipse/sw360/rest/resourceserver/component/ComponentResourceProcessor.java)